wp-content/themes/core/blocks/tribe/copyright/editor.pcss

```diff
@@ -0,0 +1,11 @@
+/* -----------------------------------------------------------------------
+ * Copyright Block
+ * ----------------------------------------------------------------------- */
+
+.wp-block-tribe-copyright {
+
+	p {
+		margin-top: 0;
+		font-size: var(--font-size-10);
+	}
+}
```

Comment:
❓ If we're including this block by default can we set this in `theme.json` instead? One less file to keep track of.

Reply:
> ❓ If we're including this block by default can we set this in `theme.json` instead? One less file to keep track of.

+1